.github/workflows/merge-queue.yml

```
@@ -3,8 +3,91 @@ name: Validate code in the merge queue (post merge)
+      image: public.ecr.aws/m3u4c4h9/island-is/actions-runner-public:latest
+    needs:
+      - prepare
+    env:
+      AFFECTED_PROJECTS: ${{ matrix.projects }}
+      MAX_JOBS: 1
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.prepare.outputs.TEST_CHUNKS) }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)